games-arcade/cdogs-sdl/cdogs-sdl-0.10.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake xdg
+
+DESCRIPTION="Classic overhead run-and-gun game"
+HOMEPAGE="https://cxong.github.io/cdogs-sdl/"
+SRC_URI="https://github.com/cxong/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+ BSD-2 CC0-1.0 CC-BY-3.0 CC-BY-SA-3.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	dev-python/protobuf-python
+	media-libs/libsdl2[haptic,opengl]
+	media-libs/sdl2-image[png]
+	media-libs/sdl2-mixer[vorbis,wav]
+	net-libs/enet:1.3="
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	rm -f doc/{COPYING.,LICENSE.,license.,README_DATA.md}* || die
+
+	# Disable cdogs-sdl-editor configure
+	sed -i -e '/add_subdirectory(cdogsed)/d' src/CMakeLists.txt || die
+	# Disable cdogs-sdl-editor build
+	sed -i -e 's/add_executable(cdogs-sdl-editor/add_executable(cdogs-sdl-editor EXCLUDE_FROM_ALL/' src/CMakeLists.txt || die
+	# Disable cdogs-sdl-editor install
+	sed -i -e 's/cdogs-sdl-editor${EXE_EXTENSION}/cdogs-sdl-editor${EXE_EXTENSION} CONFIGURATIONS Debug/' CMakeLists.txt || die
+
+	cmake_src_prepare
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DCDOGS_DATA_DIR="${EPREFIX}/usr/share/${P}/"
```

Comment:
`/usr/share/${P}/` as DATA_DIR looks legit, but why not just `/usr/share/${PN}/`?

Reply:
I figured different versions might have different data, but I suppose that isn't really the case in this package. I've changed it to `/usr/share/${PN}/` then.

Reply:
I'm hoping to make Portage not rewrite unmodified files to reduce unnecessary wear and possibly get a speed boost. Keeping these files in the same place will help there.

Reply:
Excellent, thanks!